### PR TITLE
Migrate notebooks to Quarto

### PR DESCRIPTION
**Pull request by mavam**
This PR migrates our notebooks to Quarto.

### :memo: Reviewer Checklist

Review this pull request commit-by-commit, as there is some poetry churn. Ensure the following items:

- [ ] Make sure that `make` or `yarn start` run fine in `web/`

### CODE REVIEW

**Comment by rdettai** [comment]
I couldn't run `make` in `web/` because of an R dependency issue:
```
Installing dependencies from lock file

No dependencies to install or update
Installing IPython kernel spec
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
trying URL 'http://cran.us.r-project.org/src/contrib/pak_0.3.1.tar.gz'
Content type 'application/x-gzip' length 120022 bytes (117 KB)
==================================================
downloaded 117 KB

* installing *source* package ‘pak’ ...
** package ‘pak’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** data
*** moving datasets to lazyload DB
**** building pak dependency data, this can take several minutes
**** deps data (23): ..........10..........20...23
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (pak)

The downloaded source packages are in
        ‘/tmp/RtmpxXRd0e/downloaded_packages’
Loading required package: pak
Error: 
! error in pak subprocess
Caused by error in `loadNamespace(x)`:
! there is no package called ‘pkgdepends’
---
Backtrace:
1. pak::local_install_dev_deps(upgrade = FALSE)
2. pak:::remote(function(...) { …
3. err$throw(res$error)
---
Subprocess backtrace:
1. base::withCallingHandlers(cli_message = function(msg) { …
2. get("local_install_dev_deps_make_plan", asNamespace("pak"))(...)
3. base::loadNamespace(x)
4. base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
5. base::withOneRestart(expr, restarts[[1L]])
6. base::doWithOneRestart(return(expr), restart)
7. base::stop(cond)
8. global (function (e) …
Execution halted
```

**Comment by mavam** [comment]
Unfortunately I can't reproduce this. Can you provide some info about your platform?

**Comment by mavam** [comment]
Black tried to lint the `.Renviron` file:

```
--- a/web/.Renviron
+++ b/web/.Renviron
@@ -1 +1 @@
-R_LIBS=".rlib"
+R_LIBS = ".rlib"
```

We do this in CI:

```bash
          git diff --name-only --no-color --diff-filter=ACM $(git merge-base origin/master HEAD) |
            grep -v '\(\.json\|\.csv\|\.ipynb\|\.hpp\.in\|\.ref\|\.example\|\.txt\|\.lock\|\.js\)$' |
            2>/dev/null xargs black || true
```

Any ideas how to add that file there? Just put the string `Renviron` in there?

File: .github/workflows/style-check.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
           # Note: black fails when it doesn't have to do anything.
           git diff --name-only --no-color --diff-filter=ACM $(git merge-base origin/master HEAD) |
             grep -v '\(\.json\|\.csv\|\.ipynb\|\.hpp\.in\|\.ref\|\.example\|\.txt\|\.lock\|\.js\)$' |
+            grep -v '\(Renviron\)' |
```
Comment (dominiklohmann) [inline .github/workflows/style-check.yaml:40]:
```suggestion
            grep -v '\(\.Renviron\)$' |
```

**Review by dominiklohmann — COMMENTED** [review]
I can confirm that as of [c194333](https://github.com/tenzir/vast/pull/2603/commits/c19433318b9ff3850cd2b48daabd6c40665101ab) this works nicely for me locally. I'll leave the final approval to @rdettai.

**Review by rdettai — APPROVED** [review]
If it passes CI it is fine by me. The next step is to provide a docker environment to make the run reproducible anyway.

**Pull request merged**